extractor/src/main/java/org/schabi/newpipe/extractor/Page.java

```diff
@@ -0,0 +1,45 @@
+package org.schabi.newpipe.extractor;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.List;
+
+public class Page implements Serializable {
+    private final String url;
+    private final List<String> ids;
+    private final HashMap<String, String> cookies;
```

Comment:
This should be a Map and not HashMap, same for the contructor

Reply:
You're right

Reply:
@XiangRongLin: I just fixed it.

Off-topic: once Android Studio 4 is released, we could start using streams like you wanted.

Reply:
The constructors on line 23 and 31 are still using the HashMap
https://github.com/TeamNewPipe/NewPipeExtractor/pull/314/files#diff-ba49925f61d52432cc3e52fa23360e0eR23-R33